src/eterna/mode/PoseEdit/PoseEditMode.ts

```diff
@@ -158,6 +159,24 @@ export default class PoseEditMode extends GameMode {
         this._toolbar.pairSwapButton.clicked.connect(() => this.onSwapClicked());
         this._toolbar.hintButton.clicked.connect(() => this.onHintClicked());
 
+        this._toolbar.nucleotideFindButton.clicked.connect(() => {
+            if (this._nucleotideFinderRef.isLive) {
+                return;
+            }
+
+            const {panel, positionUpdater} = NucleotideFinder.create({
```

Comment:
Should this be handled something like how this is instead? https://github.com/eternagame/EternaJS/blob/master/src/eterna/mode/PoseEdit/SubmitPoseDialog.ts

Reply:
Indeed! Works great